orchestrator/orchestrator.go

```diff
@@ -81,7 +84,8 @@ func NewGravityOrchestrator(
 		batchRequesterLoopDuration: batchRequesterLoopDuration,
 		ethBlocksPerLoop:           uint64(ethBlocksPerLoop),
 		bridgeStartHeight:          uint64(bridgeStartHeight),
-		priceFeeder:                priceFeeder,
+		coinGecko:                  priceFeeder,
```

Comment:
This is a bit confusing - especially since `priceFeeder` is an object defined in `oracle.go` as something that determines prices, while coingecko exclusively is used to convert eth addresses -> symbol. Maybe we should rename this to something like `symbolRetriever`?

Reply:
@adamewozniak it became a bit weird to have one interface in each package with the same method, what do you think?

Reply:
If we don't need an interface here / we're not mocking anything, no need to add one - I just meant that this line specifically was confusing, since something else named priceFeeder was in here at the same time, and coingecko isn't _really_ feeding prices
```golang
coinGecko:                  priceFeeder,
```